src/core_plugins/kibana/public/visualize/editor/agg.html

```diff
@@ -1,5 +1,5 @@
 <!-- header -->
-<div class="vis-editor-agg-header">
+<div class="visEditorAggHeader">
```

Comment:
Don't forget to revert these class names back to the original snake-case versions in all of the markup too.